src/partials/messenger.ts

```diff
@@ -1291,6 +1330,12 @@ class NavigationController {
         return this.notificationService.getDndModeSimplified(conversation);
     }
 
+    /**
+     * Return true if the minimal user interface is selected.
+     */
+    private isMinimalUserInterface(userInterface: UserInterface): boolean {
```

Comment:
Since this is the only case where the type alias for `UserInterface` is being used, I'd stick with the fully qualified name:

```suggestion
    private isMinimalUserInterface(userInterface: threema.UserInterface): boolean {
```

Then you can remove the alias (`import UserInterface = threema.UserInterface`) from line 52.